### PR TITLE
SAK-45875 SAMIGO - Statistics for calculated and point value is 0

### PR DESCRIPTION
**Pull request by bgarciaentornos**
Most of the changes are just tabbing (the whole file is messed up though). The main changes I've introduced are the delegate calls, which is what's being done on the other listeners where calculated answers are... calculated.

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java [hunk 1]
```diff
@@ -1702,70 +1702,58 @@ private void getTFMCScores(Map publishedAnswerHash, List scores, HistogramQuesti
...
+		results.put(CORRECT, Integer.valueOf(0));
+		results.put(INCORRECT, Integer.valueOf(0));
+
+		Map<Integer, String> answersMap = new HashMap<>();
+		int total = 0;
+		for (ItemGradingData score : scores) {
+			delegate.extractCalcQAnswersArray(answersMap, item, score.getAssessmentGradingId(), score.getAgentId());
+			if (score.getAutoScore() != null) {
+				total++;
+				if (delegate.getCalcQResult(score, item, answersMap, 1)) {
```
Comment (jesusmmp) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java:1721]: I think this should be: 

{code}
if (delegate.getCalcQResult(score, item, answersMap, 1)) {
{code}

**Comment by jesusmmp** [comment]
This patch isn't working...

**Comment by bgarciaentornos** [comment]
This only covers one formula only cases, but as Jesus says, it needs to be updated to cover >1 formulas. Thanks

**Comment by bgarciaentornos** [comment]
Added a commit with the changes provided by @jesusmmp

**Review by jesusmmp — APPROVED** [review]
LGTM!

**Review by mpellicer — APPROVED** [review]

**Pull request merged**